doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -191,7 +191,6 @@ The values in this table may change at any time. You should not rely on them for
 any kind of capacity planning.
 {{< /warning >}}
 
-<!-- RELATION_SPEC mz_internal.mz_cluster_replica_sizes -->
 | Field                  | Type        | Meaning                                                                                                                                                      |
```

Comment:
Can you move this table description into `mz_catalog.md`? Then running `ci/test/lint-catalog-docs.sh` should result in the test from `autogenerated/mz_internal.slt` getting re-created in `autogenerated/mz_catalog.slt`

Reply:
We don't want the docs change to reflect until the code is released tho right? If I moved it in this PR, wouldn't the docs would change at merge time?
Was thinking I'd have a separate PR to move the table description to `mz_catalog.md` that I merge the release goes out, but definitely want a sanity check there.

Reply:
Chatted offline with @chaas, I totally agree we don't want the docs to be updated before the change goes out, so the PR as-is looks good to me. Cara will follow up and update the docs once the release goes out next week.

Reply:
> We don't want the docs change to reflect until the code is released tho right? If I moved it in this PR, wouldn't the docs change at merge time?

The good news is that you can merge this along with the docs if you use the `warn-if-unreleased` shortcode, as shown [here](https://github.com/MaterializeInc/materialize/tree/main/doc/user#adding-documentation-for-new-features).